core/src/main/java/io/forestframework/core/http/routing/PlainHttpRoutingMatchResult.java

```diff
@@ -140,6 +142,7 @@ public HttpStatusCode getStatusCode() {
                         .thenComparing(Routing::getType)
                         .thenComparing(Routing::getPath)
                         .thenComparing(Routing::getRegexPath)
+                        .thenComparing(routing -> routing.getHandlerMethod().toString())
```

Comment:
We can just:

```
ROUTING_COMPARATOR 
 = Comparator.comparing(Routing::getOrder).thenComparing(routing -> routing.getHandlerMethod().toString());
```

No need to compare others.

Reply:
yep, that's much better.